eslint.config.mjs

```diff
@@ -159,6 +161,31 @@ export default tseslint.config([
       // We violate this rule in a lot of places. We'll turn it off for now.
       'vitest/no-identical-title': ['off'],
 
+      // Use the recommended rules for HTML.
+      ...Object.fromEntries(
+        Object.keys(html.rules).map((value) => ['@html-eslint/' + value, 'error']),
```

Comment:
For the sake of discussion, did you consider doing `flat/recommended` (https://html-eslint.org/docs/getting-started#minimal-configuration) and then enabling extra rules we care about vs. enabling everything and then disabling a bunch below?